Project.toml

```diff
@@ -33,6 +33,6 @@ Distributions = "0.16, 0.17, 0.18, 0.19"
 RDatasets = "0.5, 0.6"
 Reexport = "0.1, 0.2"
 SpecialFunctions = "0.6, 0.7"
-StatsBase = ">= 0.30"
+StatsBase = "0.30"
```

Comment:
So with each new version of StatsBase you'll have to expand this?

Reply:
Yes, since this is considered as a major release. See https://github.com/JuliaRegistries/General/pull/227 and https://github.com/JuliaRegistries/General/pull/256.